conf/scores.d/rbl_group.conf

```diff
@@ -142,16 +142,21 @@ symbols = {
 
     "RBL_SEM" {
         weight = 1.0;
-        description = "Address is listed in Spameatingmonkey RBL";
+        description = "From address is listed in Spameatingmonkey RBL";
     }
 
     "RBL_SEM_IPV6" {
         weight = 1.0;
-        description = "Address is listed in Spameatingmonkey RBL (ipv6)";
+        description = "From address is listed in Spameatingmonkey RBL (IPv6)";
     }
 
     "RBL_VIRUSFREE_BOTNET" {
         weight = 2.0;
-        description = "Source IP is listed in virusfree.cz BL";
+        description = "From address is listed in virusfree.cz BL";
+    }
+
+    "RBL_NIXSPAM" {
+        weight = 4.0;
```

Comment:
This score seems to be too high. Can we start with a lower default weight for it?

Reply:
In the pull request, the score is the same as for Spamhaus XBL. Personally, I consider NiX spam to be similar (very low FP, reported IPs are highly unwanted, etc.) so I propose this score.

What score would you prefer?